docs/clients/multi_turn_angle_control.rst

```diff
@@ -126,72 +126,78 @@ Type ID 59 | Multi-turn Angle Controller
+		+--------+-------------------------------------+----------------+-----------+-------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+		| 33     | low_power_hold_allowed_target_error | get, set, save | float     | :math:`\text{rad}`                        | The allowed target error (how close you must get to the target before transitioning to brake) when using a low power angle hold.                                                                                                                                                                                                                   |
+		+--------+-------------------------------------+----------------+-----------+-------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+		| 34     | low_power_hold_max_brake_error      | get, set, save | float     | :math:`\text{rad}`                        | The allowed brake error (how far you must get from the target before transitioning to actively holding the position) when using a low power angle hold.                                                                                                                                                                                            |
+		+--------+-------------------------------------+----------------+-----------+-------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+		| 35     | ctrl_angle_low_power                | set            | float     | :math:`\text{rad}`                        | Command a low power hold at the given angle.                                                                                                                                                                                                                                                                                                       |
```

Comment:
instead of "at the given angle" should it be target/stow angle?

Reply:
I misread, my bad. I still think it could be clearer if you say "hold at a specified angle" instead of "the given angle".